apps/web/modules/bookings/views/bookings-single-view.getServerSideProps.tsx

```diff
@@ -174,7 +175,27 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     );
   };
 
+  const checkIfUserIsTeamMember = async (userId?: number | null) => {
+    if (!userId) return false;
+
+    const teamId = eventType.team?.id;
+    if (teamId) {
+      const teamMemberResult = await isTeamMember(userId, teamId);
+      if (teamMemberResult) return true;
+    }
+
+    const parentTeamId = eventType.parent?.teamId;
+    if (parentTeamId) {
+      const parentTeamMemberResult = await isTeamMember(userId, parentTeamId);
+      if (parentTeamMemberResult) return true;
+    }
+
+    return false;
+  };
```

Comment:
Can we see if there is some existing logic somewhere in app that already checks if the user is event's team's member? I prefer to reuse that and it must be there already